client/scss/components/_forms.scss

```diff
@@ -33,6 +33,17 @@
   font-size: inherit;
 }
 
+.form .report {
```

Comment:
@PaarthAgarwal  I have refined the approach to avoid these styles.

Also - I see that these styles are in the components form  style file (which is for form fields) and not really for form submissions.